### PR TITLE
Kill 3 line select location dialog.

### PR DESCRIPTION
**Pull request by bytescream**
Standard is to have one line that opens up to more than one line (variable) where the first line, if default, is something like "Choose a station." The default value must be an entry with a blank 'value' so the error message is displayed when they have not chosen a station. But since the select initial value comes from local storage, we need a test for blank that sets the select value to the default on no stored data.

When testing be sure to clear local data storage. (F12->Application->Local Storage->File://)

### CODE REVIEW

**Comment by bytescream** [comment]
Always wanted that fixed. If I contribute nothing else...

**Review by bytescream — COMMENTED** [review]
Didn't notice that Twisted... was still at the start of the Twin Sisters OPTION. I only added the 's' on the end. I just now made it match the[ #10 ](https://github.com/nojronatron/Bigfoot-Bib-Report-WL-Form/tree/jr/fix-twinsisters-select) mod.

File: Bigfoot Bib Report Initial.html [hunk 1]
```diff
@@ -1018,7 +1018,7 @@
                             <option value="CB_Council Bluff">Council Bluff</option>
                             <option value="CH_Chain of Lakes">Chain of Lakes</option>
                             <option value="KT_Klickitat">Klickitat</option>
-                            <option value="TS_Twin Sisters">Twisted Sisters</option>
+                            <option value="TS_Twin Sisters">Twin Sisters</option>
```
Comment (nojronatron) [inline Bigfoot Bib Report Initial.html:1021]: Thanks for catching this. I'll merge in the related PR first then handle the merge conflict so the end result will be "Twin Sisters".

File: Bigfoot Bib Report Initial.html [hunk 2]
```diff
@@ -46,7 +46,11 @@
             var storedAddress = localStorage.getItem('RTrackeradd');
             document.forms[0].address.value = storedAddress;
             var storedLocation = localStorage.getItem('RTrackerloc');
-            document.forms[0].Location.value = storedLocation;
+            if (storedLocation != "") {
```
Comment (nojronatron) [inline Bigfoot Bib Report Initial.html:49]: Another good catch. I'll test this today before I merge it in.

**Review by nojronatron — APPROVED** [review]
Passes local testing: blank, cached, cached again with msg number increments.

**Pull request merged**